source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -101,15 +102,23 @@ device_policy Class
 An object of the ``device_policy`` type is associated with a ``sycl::queue`` that is used
 to run algorithms on a SYCL device. When an algorithm runs with ``device_policy``
 it is capable of processing SYCL buffers (passed via ``oneapi::dpl::begin/end``),
-data in the host memory and data in Unified Shared Memory (USM), including USM device memory.
-Data placed in the host memory and USM can only be passed to oneDPL algorithms
+data in the host memory and data in Unified Shared Memory (USM), including device USM.
+Data placed in the host memory and USM can be passed to oneDPL algorithms
 as pointers and random access iterators. The way to transfer data from the host memory
 to a device and back is unspecified; per-element data movement to/from a temporary storage
 is a possible valid implementation.
 
 The ``KernelName`` template parameter, also aliased as ``kernel_name`` within the class template,
 is to explicitly provide a name for SYCL kernels executed by an algorithm the policy is passed to.
 
+The ``device_policy`` type is copy constructible, copy assignable, move constructible and move assignable.
+A policy instance constructed as or assigned a copy of another instance is associated with
+a ``sycl::queue`` that compares equal, as defined by `SYCL`_, to the queue of that other instance.
+A policy instance constructed as or assigned a move of another instance is associated with the queue
+of that other instance without copying it. It is unspecified whether the original policy is associated
+with any queue after the move. [*Note*: The predefined ``dpcpp_default`` policy object is immutable
```

Comment:
I think you can write a **Note** as it's done here: https://github.com/uxlfoundation/oneAPI-spec/blob/main/source/elements/oneDPL/source/index.rst

Reply:
Thanks for reviewing Andrey.

For a short clarification that depends on the context of the preceding wording an "inlined" note (similar to what the C++ standard uses) seems better to me. I will think more of it though.

Reply:
Thanks. It looks nicer now